ghost/payments/lib/payments.js

```diff
@@ -214,8 +214,9 @@ class PaymentsService {
             stripe_product_id: product.id,
             currency,
             interval: cadence,
-            amount
-        }).query().select('stripe_price_id');
+            amount,
+            active: 1
```

Comment:
@SimonBackx Gotten a bit confused here but should this be `true` or `1` here ?

Reply:
Wasn't sure about the knex/bookshelf boolean mapping. But if you think true also works, then it should be fine 😊
```suggestion
            active: true
```

Reply:
Yea think we use `true` usually but even i am confused tbh which is right/ideal. If `active:true` works locally then lets go with that as that is a better representation